src/code-block.js

```diff
@@ -15,7 +15,10 @@ const preload = (content, resourcePath) => {
 
   codeBlocks.each((idx, elem) => {
     const $elem = $(elem);
-    const rawContent = $elem.html().toString();
+    let rawContent = $elem.html().toString();
+    if (shared.codeBlockLanguagesWithGenerics.includes($elem.attr('languageType'))) {
```

Comment:
Is there ever a case where these languages would want a closing tag for any reason at all?  i.e Would Java or Csharp ever contain a string template of html?

Reply:
Not a concern here.

Reply:
👍